mysql-test/suite/period/r/delete.result

```diff
@@ -0,0 +1,270 @@
+ERROR HY000: PERIOD othertime is not found in table
+delete from t for portion of system_time from '2000-01-01' to '2018-01-01';
+ERROR HY000: PERIOD `SYSTEM_TIME` is not allowed here
+create or replace table t (id int, str text, s date, e date,
+period for apptime(s,e));
+insert into t values(1, 'data', '1999-01-01', '2018-12-12');
+insert into t values(1, 'other data', '1999-01-01', '2018-12-12');
+insert into t values(1, 'deleted', '2000-01-01', '2018-01-01');
+delete from t for portion of apptime from '2000-01-01' to '2018-01-01';
+show warnings;
+Level	Code	Message
+select * from t;
+id	str	s	e
+1	data	1999-01-01	2000-01-01
+1	other data	1999-01-01	2000-01-01
+1	data	2018-01-01	2018-12-12
+1	other data	2018-01-01	2018-12-12
+drop table t1;
+# SQL17 Case 15.7-8.b.i
+# If the column descriptor that corresponds to the i-th field of BR
+# describes an identity column, a generated column, a system-time period
+# start column, or a system-time period end column, then let V i be
+# DEFAULT.
+# auto_inrement field is updated
+create or replace table t (id int primary key auto_increment, s date, e date,
+period for apptime(s, e));
+insert into t values (default, '1999-01-01', '2018-12-12');
+delete from t for portion of apptime from '2000-01-01' to '2018-01-01';
+select * from t;
+id	s	e
+2	1999-01-01	2000-01-01
+3	2018-01-01	2018-12-12
+truncate t;
+# same for trigger case
+insert into t values (default, '1999-01-01', '2018-12-12');
+delete from t for portion of apptime from '2000-01-01' to '2018-01-01';
+select * from t;
+id	s	e
+2	1999-01-01	2000-01-01
+3	2018-01-01	2018-12-12
+select * from log_tbl;
+log
+>DEL: 1999-01-01, 2018-12-12
+<DEL: 1999-01-01, 2018-12-12
+>INS: 1999-01-01, 2000-01-01
+<INS: 1999-01-01, 2000-01-01
+>INS: 2018-01-01, 2018-12-12
+<INS: 2018-01-01, 2018-12-12
+# generated columns are updated
+create or replace table t (s date, e date,
+xs date as (s) stored, xe date as (e) stored,
+period for apptime(s, e));
+insert into t values('1999-01-01', '2018-12-12', default, default);
+select * from t;
+s	e	xs	xe
+1999-01-01	2018-12-12	1999-01-01	2018-12-12
+delete from t for portion of apptime from '2000-01-01' to '2018-01-01';
+select * from t;
+s	e	xs	xe
+1999-01-01	2000-01-01	1999-01-01	2000-01-01
+2018-01-01	2018-12-12	2018-01-01	2018-12-12
+truncate t;
+# same for trigger case
+insert into t values('1999-01-01', '2018-12-12', default, default);
+delete from t for portion of apptime from '2000-01-01' to '2018-01-01';
+select * from t;
+s	e	xs	xe
+1999-01-01	2000-01-01	1999-01-01	2000-01-01
+2018-01-01	2018-12-12	2018-01-01	2018-12-12
+select * from log_tbl;
+log
+>DEL: 1999-01-01, 2018-12-12
+<DEL: 1999-01-01, 2018-12-12
+>INS: 1999-01-01, 2000-01-01
+<INS: 1999-01-01, 2000-01-01
+>INS: 2018-01-01, 2018-12-12
+<INS: 2018-01-01, 2018-12-12
+# system-time columns are updated
+create or replace table t (s date, e date,
+period for apptime(s, e)) with system versioning;
```

Comment:
well, it was the case with easy to check by an eye result, but it couldn't handle transactions, because it uses `@timestamp` variable feature.

I dropped it in favor of generic case